WordPress/Classes/Services/ReaderSearchSuggestionService.swift

```diff
@@ -30,71 +40,35 @@ import CocoaLumberjack
     ///
     /// - Returns: A matching search phrase or nil.
     ///
-    @objc func findSuggestionForPhrase(_ phrase: String) -> ReaderSearchSuggestion? {
+    private func findSuggestionForPhrase(_ phrase: String, in context: NSManagedObjectContext) -> ReaderSearchSuggestion? {
```

Comment:
Nitpick / Out of scope convention question:

This method use `findSuggestionForPhrase(_ phrase: String, ...` a similar method in a different file from this diff use `lookup(withBlogID blogID: String, ...`. The style of the two is inconsistent.

I personally prefer the latter, but I think the [Swift API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/) would recommend using the former, as the argument is part of a [prepositional phrase](https://en.wikipedia.org/wiki/Adpositional_phrase#Prepositional_phrases).

![image](https://user-images.githubusercontent.com/1218433/215396825-62388d76-e7c7-4620-97e5-c8d2dae18fd8.png)

"For" and "with" are both prepositions, ChatGPT told me so:

<img width="416" alt="Screenshot 2023-01-30 at 4 46 46 pm" src="https://user-images.githubusercontent.com/1218433/215397142-f556adc9-6fc1-40b0-9172-351e19dd1778.png">

To be honest, I feel like breaking from the guidelines in this case. I find more value in not having to think whether to name the first parameter or not vs. using what might be better English grammar in the API names.

What do you think?

Reply:
> I think the [Swift API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/) would recommend using the former

If I understand it correctly, using the former is a case of exception where there are two or more arguments? i.e. `findSuggestionFor(phrase: "hello", author: "Tony")`

The signature `findSuggestion(forPhrase: "hello", in: context)` looks pretty nature to me, so I've changed it in https://github.com/wordpress-mobile/WordPress-iOS/pull/20010/commits/754dae1ac1b86748095c89460f40cade053fcbdf